package.json

```diff
@@ -17,19 +17,19 @@
   "dependencies": {
     "iter-tools": "^1.4.1",
     "lodash": "^4.17.11",
-    "micromatch": "^3.1.4",
```

Comment:
I don't think this dependency is needed here at all,
it is used in `obj-path-expression-parser`,
so maybe we can remove it from this package